services/user/Fractal/doc/fractal-service.wit

```diff
@@ -0,0 +1,78 @@
+interface types {
+    @since(feature = scheduled-meetings)
+    record meeting-config {
+        // @velua, you spent plenty of time on this for the Psinq UI. Would you fill in the minimal set of details we determined were necessary for meeting schedule?
+    }
+
+    record fractal {
+        service: account, // Primary key
+        created-at: timepointsec,
+        name: string,
```

Comment:
also part of profile service (and we proposed noting things that will come from the profile service as comments)